lib/rules/require-padding-newlines-before-line-comments.js

```diff
@@ -83,13 +83,9 @@ module.exports.prototype = {
         var allowFirstAfterCurly = this._allowFirstAfterCurly;
 
         file.iterateTokensByType('Line', function(comment) {
-            if (comment.loc.start.line === 1) {
```

Comment:
What do you feel like the expected behavior here? I would expect a blank line between the shebang and comment.